packages/components/src/List/Toolbar/ColumnChooserButton/index.js

```diff
@@ -0,0 +1,16 @@
+import ColumnChooserButton from './ColumnChooserButton.component';
+import ColumnChooser from './ColumnChooser';
+import { mergedColumnsChooser } from './service';
+import { useColumnChooserManager } from './hooks';
+
+export default ColumnChooserButton;
+
+const columnChooserService = {
+	utils: { mergedColumnsChooser },
```

Comment:
The idea but not sure if it's a good one :) was to create a constant hierarchy, so we can have utils, or selectors etc in service.
No objection at all to flat it. Your call.

Reply:
There won't be any selector as it's component states. Here it will be only utility services. I think you messed your mind with modules which have those kind of services. Here it's a component service, not module.
I propose to flatten, to ease the usage

Reply:
You miss my point.